client/src/components/DisplayTable.js

```diff
@@ -20,7 +20,7 @@ export class DisplayTable extends React.Component {
       unsorted_initial,
     } = props;
 
-    const { sort_values } = _.first(rows);
+    const { sort_values } = !_.isEmpty() && _.first(rows);
```

Comment:
Huh, thought trying to destructure `false` would possibly give an error, but at lest in the console this works. Learn something new every day!